rust-src/concordium_base/src/updates.rs

```diff
@@ -426,6 +426,8 @@ pub struct AuthorizationsV1 {
     pub cooldown_parameters: AccessStructure,
     /// Keys for changing the lenghts of the reward period.
     pub time_parameters:     AccessStructure,
+    /// Keys for creating a protocol level token
+    pub create_plt:          AccessStructure,
```

Comment:
Shouldn't this be an Option?
```suggestion
    pub create_plt:          Option<AccessStructure>,
```

Reply:
I am not sure this part. I used the same approach as in other `AccessStructures` - make it not optional, use `require()?` for when converting from generated types.
Where can I find out what is the correction approach from the business point of view?

Reply:
Thomas is right. It should be an Option, as we have discussed as well